tool/planet/environment.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gravitational/planet/lib/box"
+	"github.com/gravitational/planet/lib/constants"
+	"github.com/gravitational/planet/lib/utils"
+
+	etcdconf "github.com/gravitational/coordinate/config"
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
+)
+
+func updateEnvironment(kvs map[string]string) error {
+	log.WithField("kvs", kvs).Info("Update environment.")
+
+	env, err := box.ReadEnvironment(ContainerEnvironmentFile)
+	if err != nil {
+		return trace.Wrap(err, "failed to read cluster environment file")
+	}
+
+	for k, v := range kvs {
+		env.Upsert(k, v)
+	}
+
+	err = utils.SafeWriteFile(ContainerEnvironmentFile, env, constants.SharedReadMask)
```

Comment:
Do the changes to the container-environment file persist between planet restarts / reboots / upgades/rollbacks? I think they do but I always forget...

Reply:
That's another good point - they don't (I think you know they don't) - each time planet starts, it would recreate the environment file.
I see a couple of options:
  - add back direct ConfigMap access to planet so it could pull actual / previous values and update the file accordingly
  - update the planet's configuration package on each envar update operation. This will keep planet unaware of the environment variables setup in kubernetes (not sure to advantage though).